.chloggen/export-logs-resource-info-based-configuration.yaml

```diff
@@ -0,0 +1,20 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datasetexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Allow include Logs resource info export to DataSet based on new export_resource_info_on_event configuration. Fix timestamp handling."
```

Comment:
I think this is a old stray file which should not be here.

Reply:
But I do think we need / want changelog file for this serverHost change :)

Reply:
I pushed a change which removes old and stray changelog file (7e94a1ffc36023a62909c7fe67ceca9cb9a5417f) and adds a new one for this change (8a0b4e738a5d8033fcdaab6f64b8c565021f89e5).